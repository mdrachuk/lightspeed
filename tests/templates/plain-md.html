<html lang="en">
<head><title>{{ markdown.title }}</title></head>
<body>
{{ markdown.html }}
</body>
</html>