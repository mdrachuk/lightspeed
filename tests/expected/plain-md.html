<html lang="en">
<head><title>None</title></head>
<body>
<h1 id="this-is-a-test">This is a test</h1>
<p>One.</p>
<h2 id="second-heading">Second heading</h2>
<h3 id="third-heading">Third heading</h3>
<h4 id="4th-heading">4th heading</h4>
<h5 id="5th-heading">5th heading</h5>
<h5 id="6th-heading">6th heading!</h5>

</body>
</html>